src/abseil-cpp/gen_build_yaml.py

```diff
@@ -27,4 +27,4 @@
     build['build_system'] = []
     build['language'] = 'c'
     build['secure'] = False
-print(yaml.dump({'libs': builds}))
+print((yaml.dump({'libs': builds})))
```

Comment:
I guess `print((X))` should be `print(X)`? If so, please apply this for other print conversions.

Reply:
Apparently `2to3` has a flag for handling the case where you'd already `from __future__ import print_function`, which I didn't know about when running it. I had thought that maybe this was to be conservative about interpreting tuples returned out of functions, e.g., a single argument that is a tuple rather than something like `*args`, but that doesn't seem to be an issue.

Reply:
Done.